### ISSUE TITLE
Cannot find module: '@rjsf/core'.

### ISSUE THREAD

**Issue by joeraygme** [rjsf-team/react-jsonschema-form#1685]
### Description
i get this error on build and run: `Cannot find module: '@rjsf/core'. Make sure this package is installed.`

This appears to be a local problem, but hoping for some guidance.  I can create a new react typescript app and can import it no problem.  In my current app i had it importing and working yesterday.  Today in my SampleForm component i'm getting the below error.  I do have it imported in another typescript library where it doesn't break.  I'm guessing from this the error message isn't exactly correct, as I can build the app when I remove any rjsf/core dependencies from the component, and leave it in the lib. 

Things i have done:

-- `yarn remove @rjsf/core`
-- `yarn cache clear`
-- `rm -rf node_modules`
-- `yarn`
-- `yarn build`
-- created a new typescript react app and imported into a component with no problem.

##### Import Statement from SampleForm.tsx (broken)
```
import Form, { ISubmitEvent } from "@rjsf/core";

import React from "react";

import * as _ from "lodash";

import { SaveFormResponse, AppForm, DefaultIntalereForm } from "../lib/IAppForm";
```

##### Import Statement from AppForm.ts (works)
`import { ISubmitEvent } from "@rjsf/core";`

#### Expected behavior

successful build

#### Actual behavior

`./src/components/SampleForm.tsx Cannot find module: '@rjsf/core'. Make sure this package is installed.`

### Version
"@rjsf/core": "^2.0.0-alpha.5",

**Comment by epicfaace** [comment]
Oops, seems that v2.0.0-alpha.5 wasn't published properly. Fixing this now.

**Comment by epicfaace** [comment]
This should be fixed in v2.0.0-alpha.6 -- let me know if it still comes up though.
